Makefile

```diff
@@ -1,8 +1,8 @@
 __PHONY__: build build-testing build-dev build-dev-deps
 
-GO_REPO_BRANCH=master	
 CORE_REPO_BRANCH=master
 SOROBAN_TOOLS_REPO_BRANCH=main
+GO_REPO_BRANCH := $(shell ./scripts/soroban_repo_to_horizon_repo.sh $(SOROBAN_TOOLS_REPO_BRANCH))
```

Comment:
@tsachiherman , I tried this out, the script resolves to `9308f9531041` correct? when I run the docker build command with that as the git version, it looks like it doesn't resolve:

```
# v0.0.0-20221028081946-9308f9531041 from go.mod ...
$ echo  "v0.0.0-20221028081946-9308f9531041"  | cut -d \- -f 3
9308f9531041

$ docker build --platform linux/amd64 -t stellar-horizon:dev -f services/horizon/docker/Dockerfile.dev --target builder https://github.com/stellar/go.git#9308f9531041
                                                                                                                                                                       
 => ERROR [internal] load git source https://github.com/stellar/go.git#9308f9531041                                                                                                                                0.8s
------
failed to solve with frontend dockerfile.v0: failed to read dockerfile: failed to load cache key: repository does not contain ref 9308f9531041, output: ""
```

or maybe I'm not replicating correctly.

Reply:
no, I think that you've replicated it well. I don't know why I haven't seen this issue. If you could test again, you'll see that it's fixed.

The solution was to make sure to use a long-commit hash instead of the short one.

Reply:
yes, works now, awesome!